### PR TITLE
Updates for Scala 2.13

### PR DESCRIPTION
**Pull request by travisbrown**
Moved from: https://github.com/sangria-graphql/sangria-ion/pull/1

I've made a major version bump here, since this PR updates the ion-java version from 1.0.1 to 1.5.1, and since there is one breaking API change: `getListValue` now calls `toSeq`, which fits the method signature it's implementing in the `InputMarshaller` API, but not the previous more specific inferred type, which was `Buffer[Node]`. This is necessary because on 2.13 `Buffer` does not extend `scala.collection.Seq`. On 2.13, where it does, the `toSeq` will have no effect.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-version := "1.0.1-SNAPSHOT"
+mimaPreviousArtifacts := Set("org.sangria-graphql" %% "sangria-ion" % "1.0.0")
 
 description := "Sangria Amazon Ion marshalling"
 homepage := Some(url("http://sangria-graphql.org"))
 licenses := Seq("Apache License, ASL Version 2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0"))
 
-scalaVersion := "2.12.1"
-crossScalaVersions := Seq("2.11.8", "2.12.1")
+scalaVersion := "2.12.10"
```
Comment (yanns) [inline build.sbt]: You don't want to use the `2.13.0` by default?

File: build.sbt [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
+crossScalaVersions := Seq("2.11.12", "2.12.10", "2.13.0")
```
Comment (yanns) [inline build.sbt]:
```suggestion
crossScalaVersions := Seq("2.11.12", scalaVersion.value, "2.13.0")
```

File: build.sbt [hunk 3]
```diff
@@ -1,36 +1,37 @@
...
-  "org.sangria-graphql" %% "sangria-marshalling-api" % "1.0.0",
+  "org.sangria-graphql" %% "sangria-marshalling-api" % "1.0.4",
 
-  "software.amazon.ion" % "ion-java" % "1.0.1",
+  "software.amazon.ion" % "ion-java" % "1.5.1",
 
-  "org.sangria-graphql" %% "sangria-marshalling-testkit" % "1.0.0" % "test",
-  "org.scalatest" %% "scalatest" % "3.0.1" % "test"
+  "org.sangria-graphql" %% "sangria-marshalling-testkit" % "1.0.2" % Test,
+  "org.scalatest" %% "scalatest" % "3.0.8" % Test
```
Comment (yanns) [inline build.sbt:27]: 👍

**Review by yanns — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (travisbrown) [reply build.sbt]: Yes, we should be consistent here—thanks for catching!

**Pull request merged**